mkosi.clangd

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+set -eux
+
+SUBDIR="$1"
+shift
+
+if [[ -n "${MKOSI_CONFIG-}" ]]; then
+    exec mkosi-chroot env --chdir="$SRCDIR/$SUBDIR" clangd "$@"
+fi
+
+MKOSI_CONFIG="$(mkosi --directory "$(dirname "$(readlink -f "$0")")" --json summary | jq -r .Images[-1])"
+
+DISTRIBUTION="$(jq -r .Distribution <<< "$MKOSI_CONFIG")"
+RELEASE="$(jq -r .Release <<< "$MKOSI_CONFIG")"
+ARCH="$(jq -r .Architecture <<< "$MKOSI_CONFIG")"
+
+BUILDDIR="$(jq -r .BuildDirectory <<< "$MKOSI_CONFIG")"
+CACHEDIR="$(jq -r .CacheDirectory <<< "$MKOSI_CONFIG")"
+
+exec mkosi \
+    --directory "$(dirname "$(readlink -f "$0")")" \
```

Comment:
No, this does **not** work when mkosi-kernel is `Include=`'d from somewhere else! It completely throws away any config from the working directory this wrapper script was initially invoked from.

Reply:
Why do you want to `Include=` mkosi-kernel somewhere else in the first place?

Reply:
Or (this was actually my first use-case) I want to collaborate with a few other people and ensure we use a common mkosi+mkosi-kernel version. In that case I add mkosi+mkosi-kernel as git submodules and add an `Include=` statement in the top-level `mkosi.conf` file. This makes setup *way* easier for everyone involved as they just have a tree like the following:

```sh
.
├ mkosi/  # git submodule
├ mkosi-kernel/  # git submodule
├ linux/
├ src/   # for userspace progs and other misc stuff
├ other project files...
└ mkosi.conf
```